config.py

```diff
@@ -50,3 +50,11 @@
 CHATROOM_PRESENCE = os.environ.get('ROOMS', '').split() or ROOMS_TO_JOIN
 
 AUTOINSTALL_DEPS = True
+
+env_mapping = {'new-env': 'old-env'}
```

Comment:
`new-env` and `old-env` were placeholders in that example. So, you should include all the environment variables that are being renamed in this form. Example: `'BOT_ROOT': 'COBOT_ROOT'`